### PR TITLE
feat: check $XDG_HOME_CONFIG/wishlist/config.ya?ml as well

### LINKED ISSUE #73: Enhancement: look for config.yml in $HOME/.ssh
Case:
- I initiate ssh connections to other machines from a variety of working directories.
- My `$HOME/.ssh/config` file contains a mix of regular SSH connections and git repositories, which are only used by git
- In order to consistently use a default config.yml (and therefore have a connection list that excludes git repositories), I need to always pass wishlist the path to my config.yml

Suggestion: Add `$HOME/.ssh/config.[yml|yaml]` or perhaps `$HOME/.ssh/wishlist/config.[yml|yaml]` to the list of places wishlist automatically looks for configs, ahead of `$HOME/.ssh/config` and `/etc/ssh/ssh_config`.

### PR DESCRIPTION
**Pull request by caarlos0**
fixed #73

### CODE REVIEW

File: cmd/wishlist/main.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ var serverCmd = &cobra.Command{
 var configFile string
 
 func init() {
-	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "Path to the config file to use. Defaults to, in order of preference: $PWD/.wishlist/config.yaml, $PWD/.wishlist/config.yml, $HOME/.ssh/config, /etc/ssh/ssh_config")
+	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "Path to the config file to use. Defaults to, in order of preference: $PWD/.wishlist/config.yaml, $PWD/.wishlist/config.yml, $HOME/.ssh/config, $HOME/config.yaml, $HOME/config.yml, /etc/ssh/ssh_config")
```
Comment (caarlos0) [inline cmd/wishlist/main.go:123]:
```suggestion
	rootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "", "Path to the config file to use. Defaults to, in order of preference: $PWD/.wishlist/config.yaml, $PWD/.wishlist/config.yml, $HOME/config.yaml, $HOME/config.yml, $HOME/.ssh/config, /etc/ssh/ssh_config")
```

**Comment by caarlos0** [comment]
> $HOME/config.yaml feels a bit too generic, imo.

it was supposed to be $HOME/.ssh/config.ya?ml :P

**Comment by muesli** [comment]
> > $HOME/config.yaml feels a bit too generic, imo.
> 
> it was supposed to be $HOME/.ssh/config.ya?ml :P

I see. Frankly, I don't think we should start polluting others' config directories either: in the worst case ssh starts supporting a `config.yaml` itself and that would just cause trouble.

**Comment by caarlos0** [comment]
> I see. Frankly, I don't think we should start polluting others' config directories either: in the worst case ssh starts supporting a `config.yaml` itself and that would just cause trouble.

The problem is that wishlist looks at CWD, so maybe, we could instead support looking into ~/.wishlist or something like that as well...

**Pull request merged**